src/transformers/training_args.py

```diff
@@ -352,8 +352,9 @@ class TrainingArguments:
         label_names (`List[str]`, *optional*):
             The list of keys in your dictionary of inputs that correspond to the labels.
 
-            Will eventually default to `["labels"]` except if the model used is one of the `XxxForQuestionAnswering` in
-            which case it will default to `["start_positions", "end_positions"]`.
+            Will eventually default to the list of input keys that contain the word "label", except if the model used is
```

Comment:
```suggestion
            Will eventually default to the list of arguments names in `model.forward` that contain the word "label", except if the model used is
```

Reply:
In that case, should I mention `TF` and `Flax` models that use `call` and `__call__` methods instead of `forward`? Or is this more for internal usage?

Reply:
Maybe "argument names accepted by the model" then, to be framework-agnostic.